api/graphql/schema.graphql

```diff
@@ -1447,6 +1454,9 @@ type Person {
   # Project posts that this person is a member of
   projects: PostQuerySet
 
+  # This person's reactions (substitute bookmarks?)
+  reactions(first: Int, offset: Int, order: String): ReactionQuerySet
```

Comment:
In your user profile, there could be a section with everything you have reacted too, almost like a defacto bookmark section

Reply:
totally, just seems like a funny place to have this comment since this will be seen in our API docs